gazebo_ros/launch/gzclient.launch.py

```diff
@@ -0,0 +1,90 @@
+# Copyright 2019 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Launch a Gazebo client with command line arguments."""
+
+from os import environ
+
+from scripts import GazeboRosPaths
+
+from launch import LaunchDescription
+from launch.actions import DeclareLaunchArgument
+from launch.actions import ExecuteProcess
+from launch.substitutions import LaunchConfiguration
+from launch.substitutions import PythonExpression
+
+
+def generate_launch_description():
+    model, plugin, media = GazeboRosPaths.get_paths()
+
+    if 'GAZEBO_MODEL_PATH' in environ:
+        model += ':'+environ['GAZEBO_MODEL_PATH']
+    if 'GAZEBO_PLUGIN_PATH' in environ:
+        plugin += ':'+environ['GAZEBO_PLUGIN_PATH']
+    if 'GAZEBO_RESOURCE_PATH' in environ:
+        media += ':'+environ['GAZEBO_RESOURCE_PATH']
+
+    env = {'GAZEBO_MODEL_PATH': model,
+           'GAZEBO_PLUGIN_PATH': plugin,
+           'GAZEBO_RESOURCE_PATH': media}
+
+    return LaunchDescription([
+        DeclareLaunchArgument('version', default_value='false',
+                              description='Set "true" to output version information'),
+        DeclareLaunchArgument('verbose', default_value='false',
+                              description='Set "true" to increase messages written to terminal'),
+        DeclareLaunchArgument('help', default_value='false',
+                              description='Set "true" to produce gzclient help message'),
+        DeclareLaunchArgument('Timer', default_value='false',
+                              description='Set "true" to start gzclient with Timer GUI plugin'),
+        DeclareLaunchArgument('Cessna', default_value='false',
+                              description='Set "true" to start gzclient with Cessna GUI plugin'),
+        DeclareLaunchArgument('Keyboard', default_value='false',
+                              description='Set "true" to start gzclient with Keyboard GUI plugin'),
```

Comment:
Instead of exposing these 3 GUI plugins as boolean options, it would be more flexible to let the user pass the full plugin name as a string. This way, they can also choose custom plugins. What do you think?

Reply:
The problem is that one argument only takes one value.  Users cannot assign a list of plugins to an argument

Reply:
Together with @sloretz, we found out about the `shell=True` option, which lets us pass the whole command as a single string. Using that, I have some suggestions on https://github.com/ros-simulation/gazebo_ros_pkgs/commit/57216468803f5f9eba3cd585c50c9e9b75c537a6 .

The last thing I'd like to add would be `gui` and `server` arguments to `gazebo.launch.py` so that users can conditionally include `gzclient` and `gzserver`, i.e.:

`ros2 launch gazebo_ros gazebo.launch.py gui:=false`


I found documentation implying that `IncludeLaunchDescription` can be conditionally included but I didn't figure out exactly how.

* https://answers.ros.org/question/314903/ros2-optionally-launch-node-with-no-arguments/
* https://github.com/ros2/launch/blob/master/launch/doc/source/architecture.rst#base-action
* https://github.com/ros2/launch/pull/121/files

Reply:
It turns out conditional includes are not possible yet: https://github.com/ros2/launch/issues/303. We could add a note to the code pointing to that issue.

This PR looks good to me once the changes to use `shell=True` are in.